src/environment.cpp

```diff
@@ -179,6 +179,9 @@ void Environment::continueRaycast(RaycastState *state, PointedThing *result_p)
 			break; // About to go out of bounds
 		}
 
+		// for check bigSelectionBox
+		v3s16 &current_pos = state->m_iterator.m_current_node_pos;
```

Comment:
```suggestion
		// Optimization trick: to skip non-oversized selection boxes for other positions.
		const v3s16 current_pos = state->m_iterator.m_current_node_pos;
```

Reply:
Done. Our conventions say we want to copy such small vectors anyways (and we do want to).

Side note: I would consider it likely that both end up with the same machine code (at least if a const ref was used), that is, the compiler would figure this out itself.